packages/website/plugins/generated-rule-docs/nodes.ts

```diff
@@ -0,0 +1,14 @@
+import type * as mdast from 'mdast';
+import type * as unist from 'unist';
+
+export function nodeIsCode(node: unist.Node): node is mdast.Code {
```

Comment:
I suggest moving all non-tree-modifying helper functions to the same file.

Reply:
Sorry, not sure how to parse this - which helper functions are you thinking of?

Reply:
I meant these files: nodes.ts, requiredHeadings.ts, spliceChildrenAndAdjustHeadings.ts, strings.ts, types.ts. These are all small utilities and I don't entertain the idea of splitting them into 5 files well.